src/responses.js

```diff
@@ -0,0 +1,179 @@
+const { fixArraySchema, hasFileRef, getRefFromInclude, searchDataStructure } = require('./util')
+const escapeJSONPointer = require('./escape_json_pointer')
+const isEqual = require('lodash.isequal')
+const http = require('http')
+
+const parseResponseSchema = (schema, openApi3) => {
```

Comment:
Instead of explicitly passing in `openapi3`, let's use the `options` variable or `context` to derive the state